build.rs

```diff
@@ -39,15 +49,17 @@ fn main() {
     // FIXME: this exists to work around https://github.com/rust-lang/cargo/issues/10527
     println!("cargo:rerun-if-changed=crates/");
 
-    #[cfg(all(target_os = "linux", target_arch = "x86_64",))]
-    println!("cargo:rustc-cfg=enarx_with_shim");
+    if *IS_X86_64_LINUX {
+        println!("cargo:rustc-cfg=enarx_with_shim");
+    }
 
-    #[cfg(all(target_os = "linux", target_arch = "x86_64"))]
-    generate_protos();
+    if *IS_X86_64_LINUX {
+        generate_protos();
+    }
 
     #[cfg(all(target_os = "linux", target_arch = "x86_64"))]
```

Comment:
You don't want to change this cfg as well?

Reply:
No, because this is only valid when x86_64 linux is the host